parsec/core/gui/file_history_widget.py

```diff
@@ -167,9 +167,15 @@ def on_get_version_success(
         if download_limit_reached:
             self.button_load_more_entries.setVisible(False)
         self.versions_job = None
+        version_counter = 0
+        current_id: EntryID | None = None
         for author, version in versions_list:
+            if version.id != current_id:
+                version_counter += 1
+                current_id = version.id
+            version_string = f"{version_counter}.{version.version}"
```

Comment:
Why not the other way around ?
Why we have the counter first and not the version, I'm confused

Reply:
`version.version` here represents the version of the vlob.
But when a file is replaced by another one, a new entry is created so the versioning for the new entry starts back at 1.